cla_backend/apps/call_centre/tests/api/test_case_api.py

```diff
@@ -855,7 +855,7 @@ def _default_local_dt_sla_120(self):
 
     @property
     def _default_local_dt_sla_480(self):
-        return self._default_local_dt + datetime.timedelta(minutes=480)
+        return self._default_local_dt + datetime.timedelta(days=1)
```

Comment:
There are quite a few changes like these, what's the reasoning behind the timedelta change?

Reply:
Good catch. Will answer this in the PR description.